storage/redis_cluster.go

```diff
@@ -115,35 +52,38 @@ func getRedisAddrs(config config.StorageOptionsConf) (addrs []string) {
 	return addrs
 }
 
-func clusterConnectionIsOpen(cluster *RedisCluster) bool {
-
-	c := cluster.RedisController.singleton(cluster.IsCache, cluster.IsAnalytics)
-	testKey := "redis-test-" + uuid.NewV4().String()
-	if err := c.Set(cluster.RedisController.ctx, testKey, "test", time.Second).Err(); err != nil {
-		return false
-	}
-	if _, err := c.Get(cluster.RedisController.ctx, testKey).Result(); err != nil {
-		return false
-	}
-	return true
-}
-
 // Connect will establish a connection this is always true because we are
 // dynamically using redis
 func (r *RedisCluster) Connect() bool {
 	return true
 }
 
-func (r *RedisCluster) singleton() redis.UniversalClient {
+func (r *RedisCluster) singleton() RedisDriver {
 	return r.RedisController.singleton(r.IsCache, r.IsAnalytics)
 }
 
-func (r *RedisCluster) hashKey(in string) string {
+func (r *RedisCluster) check(err error, message string, opt ...interface{}) {
```

Comment:
@titpetric I don't fully get the usage of this `check` func in the handler. What was your idea / what you are trying to solve here?

Reply:
Consistency in logging the errors. Storage interface doesn't return them, some functions omit logging, etc.

Ideally this is throw away code as soon as we move forward with a LoggedStorage wrapper around Storage (BC).